presto-docs/requirements.txt

```diff
@@ -27,6 +27,7 @@ soupsieve==2.0.1
 Sphinx==2.4.4
 sphinx-material==0.0.30
 sphinxcontrib-applehelp==1.0.2
+sphinx-copybutton==0.3.1
```

Comment:
is this the latest version?

Reply:
Yes, https://github.com/executablebooks/sphinx-copybutton/releases/tag/v0.3.1 released 3 days ago

Reply:
great